influxql/iterator.gen.go.tmpl

```diff
@@ -466,7 +466,7 @@ func (itr *{{$k.name}}ParallelIterator) Close() error {
 func (itr *{{$k.name}}ParallelIterator) Next() (*{{$k.Name}}Point, error) {
 	v, ok := <-itr.ch
 	if !ok {
-		return nil, io.EOF
+		return nil, nil
```

Comment:
It seems like we should return an error if we're calling `Next()` after `Close()` has been called.